ppml/tdx/docker/trusted-big-data/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM intelanalytics/bigdl-k8s
+
+# Install python3.8
```

Comment:
organize the docker commands as follow order
ENV
ADD
RUN

may combine several RUN together to reduce the image size